aclk/aclk_rx_msgs.c

```diff
@@ -15,18 +15,52 @@ static inline int aclk_extract_v2_data(char *payload, char **data)
     return 0;
 }
 
-static inline int aclk_v2_payload_get_query(const char *payload, struct aclk_request *req)
+#define ACLK_GET_REQ "GET "
+#define ACLK_CHILD_REQ "/host/"
+#define ACLK_CLOUD_REQ_V2_PREFIX "/api/v1/"
+#define STRNCMP_CONST(str, const) strncmp(str, const, strlen(const))
+static inline int aclk_v2_payload_get_query(struct aclk_cloud_req_v2 *cloud_req, struct aclk_request *req)
 {
-    const char *start, *end;
+    const char *start, *end, *ptr;
+    char uuid_str[UUID_STR_LEN];
```

Comment:
we use `GUID_LEN + 1` for this in a lot of places in the code

Reply:
I saw that in Netdata code but I prefer this one as it comes from UUID lib directly but can change it if you really insist